cre.cpp

```diff
@@ -1257,6 +1257,30 @@ void lua_pushSegmentsFromRange(lua_State *L, CreDocument *doc, ldomXRange *range
     }
 }
 
+static int nextVisibleWordEnd(lua_State *L){
```

Comment:
**get**NextVisibleWordEnd() and **get**PrevVisibleWordEnd() , for reasons explained in https://github.com/koreader/koreader/issues/4555#issuecomment-462245085

Reply:
(I see you've done that in credocument.lua. They deserve than name here too :)